source/dotnet/wholesale-api/Edi/Validation/WholesaleServicesRequest/Rules/PeriodValidationRule.cs

```diff
@@ -107,10 +92,18 @@ public Task<IList<ValidationError>> ValidateAsync(DataHub.Edi.Requests.Wholesale
         return null;
     }
 
-    private void MustBeWithin3YearsAnd2Months(Instant periodStart, ICollection<ValidationError> errors)
+    private void MustNotBe3YearsAnd3MonthsOld(Instant periodStart, ICollection<ValidationError> errors)
     {
-        if (periodValidationHelper.IsStartDateOlderThanAllowed(periodStart, maxYears: 3, maxMonths: -2))
-            errors.Add(_startDateMustBeLessThanOrEqualTo3YearsAnd2Months);
+        // Start is within [3 year 2 months ago, now]
+        if (!periodValidationHelper.IsDateOlderThanAllowed(periodStart, maxYears: 3, maxMonths: 2))
+            return;
+
+        // Start is within [A long time ago, 3 year 3 months ago]
+        if (periodValidationHelper.IsDateOlderThanAllowed(periodStart, maxYears: 3, maxMonths: 3)
+            || periodValidationHelper.PeriodStartIs3YearsAnd3MonthsAgo(periodStart))
+        {
+            errors.Add(_startDateMustBeLessThanOrEqualTo3YearsAnd3Months);
+        }
```

Comment:
Could you not do something like this instead and the adjust the method name?

```suggestion
        if (periodValidationHelper.IsDateOlderThanAllowed(periodStart, maxYears: 3, maxMonths: 2))
        {
            errors.Add(_startDateMustBeLessThanOrEqualTo3YearsAnd3Months);
        }
```